cmd/oras/pull.go

```diff
@@ -146,9 +146,14 @@ func runPull(opts pullOptions) error {
 	copyOptions.PostCopy = func(ctx context.Context, desc ocispec.Descriptor) error {
 		name := desc.Annotations[ocispec.AnnotationTitle]
 		if name == "" {
```

Comment:
Let's align this to `PrintStatus`
```suggestion
		name, ok := desc.Annotations[ocispec.AnnotationTitle]
	        if !ok 
```

Reply:
Sure, haha